includes/class-sm-post-types.php

```diff
@@ -7,6 +7,8 @@
 
 defined( 'ABSPATH' ) or die;
 
+require_once SM_PATH . 'sermons.php';
```

Comment:
No need to require the `sermons.php` file here. `sermons.php` will require this file anyway.